Goobi/src/de/sub/goobi/helper/tasks/ExportNewspaperBatchTask.java

```diff
@@ -771,6 +771,48 @@ private static Integer positionByRank(List<DocStruct> siblings, String metadataT
         return result;
     }
 
+    /**
+     * Returns the index of the issue to insert between its siblings depending
+     * on its rank. A return value of {@code null} will indicate that no
+     * position could be determined which will cause
+     * {@link DocStruct#addChild(Integer, DocStruct)} to simply append the new
+     * child at the end.
+     *
+     * @param siblings
+     *            brothers and sisters of the child to add
+     * @param rank
+     *            rank of the child to insert
+     * @return the index position to insert the child
+     */
+    private static Integer positionByRank(List<DocStruct> siblings, Integer rank) {
```

Comment:
Why did you add this method as a static one?

Reply:
It doesn’t need access to fields of the class instance. I basically declare any method static if it is. You do not have to, I know. But why should I leave the assumption of a dependency if I know that there is no such?